File: Reverse Integer.cpp
class Solution {
public:
   int reverse(int x) {
       // Note: The Solution object is instantiated only once and is reused by each test case.
       int res = 0;
       while(x){
           res = res*10+x%10;  //no matter positive or negative
           x/=10;
       }
       return res;
   }
};

File: python/Reverse Integer.py
class Solution:
    # @return an integer
    def reverse(self, x):
        res = 0
        flag = (x >= 0)
        x = abs(x)
        while x != 0:
            res = res*10 + x%10
            x /= 10
        if flag:
            return res
        else:
            return 0-res        
            